src/main/resources/templates/user.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Messages by {{user.username}}</title>
+</head>
+<body>
+<p>
+<form action="/logout" method="post" style="display: inline">
+    <button type="submit">Logout</button>
+</form>
+<a href="/"><button>Home</button></a>{{currentUser}} is currently logged in
```

Comment:
As we get more sophisticated with these programs start making the text on the page look more like what you would see.  Like "Welcome {{currentUser}}"